src/components/Modal_Components/LinkedTransaction.jsx

```diff
@@ -7,15 +7,19 @@ import TransactionForm from "./TransactionForm";
 import "./modalStyle.css";
 
 /**
- * AddTransaction (Modal)
+ * LinkedTransaction (Modal)
+ *
+ * this modal relies on a 3rd party API.
+ *
+ * (called from ../Transaction_Components/LinkedTransactions)
  * @param {String} userID userID
  * @param {Function} open handler
  * @param {Function} handleClose handler
  * @param {Function} addTransaction handler
  *
  * @returns {*} Add Transaction Modal
  */
-export const AddTransaction = ({
+export const LinkedTransaction = ({
```

Comment:
It makes sense to rename this to match where it's being called, especially if we're not using it anywhere else.